Index: pym/_emerge/__init__.py
===================================================================
--- pym/_emerge/__init__.py	(revision 13408)
+++ pym/_emerge/__init__.py	(revision 13409)
@@ -9854,9 +9854,10 @@
 		'newline'         : 'nel',
 	}
 
-	def __init__(self, out=sys.stdout, quiet=False):
+	def __init__(self, out=sys.stdout, quiet=False, xterm_titles=True):
 		object.__setattr__(self, "out", out)
 		object.__setattr__(self, "quiet", quiet)
+		object.__setattr__(self, "xterm_titles", xterm_titles)
 		object.__setattr__(self, "maxval", 0)
 		object.__setattr__(self, "merges", 0)
 		object.__setattr__(self, "_changed", False)
@@ -10076,7 +10077,8 @@
 		else:
 			self._update(color_output.getvalue())
 
-		xtermTitle(" ".join(plain_output.split()))
+		if self.xterm_titles:
+			xtermTitle(" ".join(plain_output.split()))
 
 class ProgressHandler(object):
 	def __init__(self):
@@ -10220,7 +10222,8 @@
 		# being in a fragile state. For example, see bug #259954.
 		self._unsatisfied_system_deps = set()
 
-		self._status_display = JobStatusDisplay()
+		self._status_display = JobStatusDisplay(
+			xterm_titles=('notitles' not in settings.features))
 		self._max_load = myopts.get("--load-average")
 		max_jobs = myopts.get("--jobs")
 		if max_jobs is None:
